### PR TITLE
Feature/create_dashboard_page

### PR DESCRIPTION
**Pull request by okurosawa**
## Description / Purpose (checkpoints)
These change provide feature that User can see all users activities, how many lessons taken and how many words learned. 

These are summary what I did.
- I modified `activity_type` column in `Activities` table because of I want to use `activity_type` as string. 
- I edited `home.blade.php` that shows dashboard. 
- I created components that related to `dashboard page`.
- I added relationship and function to Model.

[*Asana task page*]
https://app.asana.com/0/1173796632051606/1174497130657823

## How To Test This PR
1. Create two accounts.
2. Login to E-Learning system and open dashboard page.
3. You can see all users activities, how many lessons taken and how many words learned.
  But now, there is no function to take lesson or follow other user. So if you want to test about show users activities, you have to insert records to `Lessons` table, `Categories` table, `Words` table, `Answers` table and `Relationships` table using MySQL client software or using SQL directly.

 **Commands**
You have to run this command on your environment because of I added migration file that modify already exists table's column.
```shell
php artisan migrate
```

 **Screenshots**
- Dashboard page if there are some users activities.
page1

- Dashboard page if there is no users activity.
page2

### CODE REVIEW

File: app/Activity.php [hunk 1]
```diff
@@ -6,5 +6,15 @@
 
 class Activity extends Model
 {
-    //
+    protected $fillable = ['user_id', 'activity_id', 'activity_type'];
+
+    public function activity_morph()
```
Comment (jeremiahC) [inline app/Activity.php]: Please use camelCase() for methods as mention on the PSR-1 (standards for php code)
https://www.php-fig.org/psr/psr-1/#43-methods

File: app/Relationship.php [hunk 2]
```diff
@@ -6,5 +6,13 @@
 
 class Relationship extends Model
 {
-    //
+    public function activities()
+    {
+        return $this->morphMany('App\Activity', 'activity');
+    }
+
+    public function following_user()
```
Comment (jeremiahC) [inline app/Relationship.php]: Please use camelCase() for methods as mention on the PSR-1 (standards for php code)
https://www.php-fig.org/psr/psr-1/#43-methods

File: app/Activity.php (hunk 1 above)
Comment (okurosawa) [reply app/Activity.php]: I modified function name `activity_morph` to `activityMorph` on these file.
- `app\Activity.php`
- `resources\views\components\user-activity.blade.php`

File: app/Relationship.php (hunk 2 above)
Comment (okurosawa) [reply app/Relationship.php]: I modified function name `following_user` to `followingUser` on these file.
- `app\Relationship.php`
- `resources\views\components\user-activity.blade.php`

**Comment by jeremiahC** [comment]
Screen Shot 2020-05-12 at 4 03 31 PM

I think you missed out this part

**Comment by okurosawa** [comment]
> Screen Shot 2020-05-12 at 4 03 31 PM
> 
> I think you missed out this part

I added this part at activities section on dashboard page.

**Review by jeremiahC — APPROVED** [review]

**Pull request merged**